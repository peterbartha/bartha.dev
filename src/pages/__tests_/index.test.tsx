import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../index';

describe('Header', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Home />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
