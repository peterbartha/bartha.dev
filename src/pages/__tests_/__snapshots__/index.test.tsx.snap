// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<div
  data-testid="hello"
>
  Hello world!
</div>
`;
